--- Fightcade/fc2-electron/resources/app/lib/main.js	2020-07-27 09:05:52.000000000 -0400
+++ main.js	2020-09-29 10:19:37.687986200 -0400
@@ -8824,7 +8824,13 @@
                         electron_1.shell.openPath(path.dirname(process.execPath)+'\\..\\ROMs');
                     }
                     if (helpers_1.isLinux()) {
-                        electron_1.shell.openPath(path.dirname(process.execPath)+'/../ROMs');
+                        fs.stat('/.flatpak-info', (error, stats) => {
+                            if (error) {
+                                electron_1.shell.openPath(path.dirname(process.execPath)+'/../ROMs');
+                            } else {
+                                electron_1.shell.openPath(path.join(process.env['XDG_DATA_HOME'], 'ROMs'));
+                            }
+                        });
                     }
                 },
             },
